tests/distro-regression/tests/bundles/focal-wallaby.yaml

```diff
@@ -13,7 +13,7 @@ applications:
     channel: wallaby/edge
   aodh-mysql-router:
     charm: ch:mysql-router
-    channel: latest/edge
+    channel: 8.0.19/edge
```

Comment:
yesterday we changed the track name from 8.0.19 to 8.0 - https://github.com/openstack-charmers/release-tools/pull/230